src/modules/betaAlpha/events/guildMemberUpdate.ts

```diff
@@ -1,27 +1,27 @@
-import { GuildMember } from "discord.js";
+import { DiscordEvent } from "discord-module-loader";
 
 import { pmdDB } from "../../..";
 import { AlphaUsers, BetaUsers } from "../../../../@types/interfaces";
 import config from "../../../config";
 
-export default async function (oldMember: GuildMember, newMember: GuildMember) {
+export default new DiscordEvent("guildMemberUpdate", async (oldMember, newMember) => {
 	const oldRoles = oldMember.roles.cache,
 		newRoles = newMember.roles.cache;
 
 	//* Member get's alpha role
 	if (!oldRoles.has(config.roles.alpha) && newRoles.has(config.roles.alpha)) {
 		if (newRoles.has(config.roles.beta)) await newMember.roles.remove(config.roles.beta);
 
-		return await pmdDB
+		return void (await pmdDB
```

Comment:
return void is no longer needed in latest version

Reply:
Resolved